src/Nethermind/Nethermind.Core/Specs/ForkActivation.cs

```diff
@@ -9,6 +9,7 @@ namespace Nethermind.Core.Specs;
 {
     public long BlockNumber { get; }
     public ulong? Timestamp { get; }
+    public ulong Activation => Timestamp ?? (ulong)BlockNumber;
```

Comment:
Even forkid uses timestamp now?

Reply:
Yes that is what will it be doing shanghai and after